dds.sh

```diff
@@ -1,6 +1,6 @@
 package: DDS
 version: "%(tag_basename)s"
-tag: "3.7.22"
+tag: "3.7.23"
```

Comment:
This is good to go already, so I split it in a different PR (https://github.com/alisw/alidist/pull/5192).